qiskit/providers/ibmq/managed/managedjob.py

```diff
@@ -111,14 +111,19 @@ def status(self) -> Optional[JobStatus]:
 
         return None
 
-    def result(self, timeout: Optional[float] = None) -> Result:
+    def result(
+            self,
+            timeout: Optional[float] = None,
+            partial: bool = False
+    ) -> Optional[Result]:
         """Return the result of the job.
 
         Args:
            timeout: number of seconds to wait for job
+           partial: If true, attempt to retrieve partial job results.
 
         Returns:
-            Result object
+            Result object or ``None`` if result could not be retrieved.
```

Comment:
Not blocking, but in the same spirit as the `IBMQJob.result()` discussion in an earlier PR, I wonder if it would not be too costly (in terms of implications and changes) to avoid having this method return `None`, for preserving the parallelism between a managed job and a IBMQJob.